readme.md

```diff
@@ -1,17 +1,31 @@
 # Construct
 
+Construct helps you to quickly setup your computer with the tools we have found helpful here at Segment.
 
-Install construct
+Construct will install the following tools:  
 
-  git clone git@github.com:pivotal-cloudops/construct.git
+  atom
+  commandline-tools
+  homebrew
+  homebrew-casks
+  iterm
+  google-chrome
+  recordit
+  slack
+  vagrant
+  virtualbox
 
-Then run it
+
+To install Construct, open your terminal and type:
+
+  `git clone git@github.com:TeresaNesteby/segment-construct.git`  
+
+Then in your terminal type:
```

Comment:
Depending on how wide your user base is you made need to tell them to cd into that directory too.